articles/native-platforms/windows-store-csharp.md

```diff
@@ -39,22 +42,7 @@ There are three options to do the integration:
 To start with, we'd recommend using the __Login Widget__. Here is a snippet of code to copy & paste on your project.
 Since we are using `await` (.NET 4.5 or greater), your method needs to be `async`:
 
-```cs
-using Auth0.SDK;
-
-var auth0 = new Auth0Client(
-	"@@account.namespace@@",
```

Comment:
Let's separate this into 2 snippets, one on top of the other. The first 3 lines are the setup. Then it's the use. Makes sense? @ntotten